common-util/constants.js

```diff
@@ -5,7 +5,7 @@ export const LAUNCH_CONTACT_URL = 'https://t.me/pahlmeyer';
 
 export const MENU_DATA = [
   { link: '/learn', text: 'Learn' },
-  // { link: '/explore', text: 'Explore' },
+  { link: '/explore', text: 'Explore' },
```

Comment:
No I don't want that back in the menu. This page is not that important to give it a menu bar section. If we want we can add it to "Resources" submenu

Reply:
Okay, I can change that 👍 Is it alright to keep the footer link or should that be removed as well? 
![image](https://github.com/user-attachments/assets/3f531dff-78e5-4eab-8e62-3d9c3703b1bf)

Reply:
I've removed Explore and put it under Resources instead